data/wanderer/wanderers start.txt

```diff
@@ -260,113 +262,67 @@ mission "Wanderers: Translation Machine"
-	`	You go to your ship to retrieve the data and give it to him upon returning. His eyes light up with excitement as he makes a copy of the card and returns the original to you.`
-		goto give
-	
-	label return
-	`You return to Eruk's workshop and give him the data card. His eyes light up with excitement as he makes a copy of the card and returns the original to you.`
-	
-	label give
-	`	"I have spent my whole life studying human culture," he says. "This is a treasure to me. And now, here is your speaking machine." He hands you a small, spherical device, with speakers and microphones mounted on it, as well as what look like camera lenses. "The device is programmed with the human and Hai languages," he says. "It is not always possible to translate perfectly between our two languages, but it will try to give you a sense of the range of meanings in what is being said."`
-	choice
-		`	"Thank you. I'm eager to try it out!"`
-			accept
-		`	"Is there anything else I need to know?"`
-	`	He humors you by demonstrating how the machine works, holding a conversation with you where he speaks in the Hai language, and the machine automatically translates for you and then translates your responses back into his language. It apparently uses its cameras to track who you are speaking to or listening to. It's an impressive piece of technology, and easy enough to use. You thank him for his help, and return to your ship.`
-		accept
-
-
-
-mission "Human Cultural Archives"
-	description "Eruk, the Hai technologist on <origin>, has agreed to give you a translation machine if you can somehow get him a copy of the human cultural data from <destination>."
-	landing
-	source "Greenwater"
-	destination "Alexandria"
-	to offer
-		has "Wanderers: Translation Machine: done"
-		not "human cultural data"
-	on offer
-		conversation "eruk cultural data"
-	on visit
-		dialog
-			`The station has a small museum gift shop that sells a copy of the entire archive on a data card. It only costs 40 credits, but because you have done a horrible job at managing your finances you do not have even that much cash on hand right now.`
-			`	Go earn some money, then return here.`
 	on complete
 		set "human cultural data"
 		payment -40
+		conversation
+			`<Library search> She directs you to the museum gift shop, where, to your surprise, you could buy the full "Human Cultural Archive" for a mere 40 credits.`
+				to display
+					not "Bankrupt at Alexandria"
+			`Now that you finally have 40 credits, you purchase the "Human Cultural Archive" that you must take to <planet>.`
+				to display
+					has "Bankrupt at Alexandria"
+	on visit
+		set "Bankrupt at Alexandria"
+		conversation
+			`<Library search> She directs you to the museum gift shop, to buy the "Human Cultural Archive" for 40 credits.`
+			`	Unfortunately, you have done such a horrible job at managing your finances you do not have even that much cash on hand right now.`
+			`	Go earn some money, then return here.`
```

Comment:
This, for example, is the data set that might be somewhat misleading if used to train the algorithm en masse.
I'm sure it contains many things that would lead to archaic language being used that'd make no sense to you.

Reply:
It won't be in an archaic language. Whatever is in there will be in the language of modern day since it is intended to be understandable by people who know only the common language.

Reply:
Long term storage of bulk data is generally done in original language, not copied from translations or translated on sight (especially scanned text).

There absolutely is almost certainly a translated explanation/summary of just about everything in that archive, but the majority of its capacity will be in hundreds of different human languages.

A specific translation database for the hundreds of different human languages all into the most common current one (which we already do have now, for multiple different current languages, even, so should definitely be even more refined in another thousand years) would be exceedingly helpful for developing translation into human from alien, regardless, though.

Reply:
My general assumption is that the "Human cultural archive" that one can buy in the gift shop is not, in fact, the sum total of the entire library in archival format, but rather a curated cross section of human culture. Most tourists, for instance, probably don't have too much interest (or capacity) to understand some random book in the original language from 500 years ago, for instance. Likewise, I strongly doubt that it has *everything* humanity has produced. To use a modern example, if someone created a "Human cultural archive" for the 20th century, I wouldn't expect it to have a copy of every episode of every TV show ever produced. Nor every book written during the time period, or every movie. Assuming space wasn't a serious limitation, I'd expect it to have something like the top ten thousand TV show episodes deemed by anthropologists, sociologists, and other experts, to be the best representations of human culture and society during the time period. If we're being generous, maybe the top hundred thousand books and the top several hundred movies. (For comparison, there were roughly 3 million books published in 2010, depending on definitions (310k traditional books, another 2.7M e-books of various kinds)). Even assuming storage densities can pack hundreds of terrabytes into that little storage device, I doubt it could hold even 0.1% of the sum of human culture. Which means filtering and selection has to take place, presumably by the cultural experts on staff.

 I think it's a lot more likely that this cultural archive is one of two things:

a) a carefully generated data file that has been processed and annotated to work with human translation software so that it can generate the most accurate possible rendering in whatever the user's language happens to be. 

b) a multi-lingual file that contains all the contained works in pre-translated media files in a whatever the most common languages are today.

Honorable mention to the third pseudo fair option: Each archive is either randomly or sequentially generated from the sum contents of the library, such that every one of them has a similar but not identical cross section of human culture. Given enough data archives, the entirety of the library could be assembled (probably millions), but the only a tiny fraction would be present on any one archive. (For instance, every archive might have, say, a loony tunes cartoon episode. But each one has a different episode.)